src/Security/Authentication/OpenIdConnect/src/LoggingExtensions.cs

```diff
@@ -61,201 +61,201 @@ static LoggingExtensions()
             _enteringOpenIdAuthenticationHandlerHandleSignOutAsync = LoggerMessage.Define<string>(
-                eventId: 14,
+                eventId: new EventId(14, "EnteringOpenIdAuthenticationHandlerHandleSignOutAsync"),
                 logLevel: LogLevel.Trace,
                 formatString: "Entering {OpenIdConnectHandlerType}'s HandleSignOutAsync.");
             _postAuthenticationLocalRedirect = LoggerMessage.Define<string>(
-                eventId: 5,
+                eventId: new EventId(5, "PostAuthenticationLocalRedirect"),
                 logLevel: LogLevel.Trace,
                 formatString: "Using properties.RedirectUri for 'local redirect' post authentication: '{RedirectUri}'.");
             _redirectToIdentityProviderHandledResponse = LoggerMessage.Define(
-                eventId: 6,
+                eventId: new EventId(6, "RedirectToIdentityProviderHandledResponse"),
                 logLevel: LogLevel.Debug,
                 formatString: "RedirectToIdentityProvider.HandledResponse");
             _invalidAuthenticationRequestUrl = LoggerMessage.Define<string>(
-                eventId: 8,
+                eventId: new EventId(8, "InvalidAuthenticationRequestUrl"),
                 logLevel: LogLevel.Warning,
                 formatString: "The redirect URI is not well-formed. The URI is: '{AuthenticationRequestUrl}'.");
             _enteringOpenIdAuthenticationHandlerHandleRemoteAuthenticateAsync = LoggerMessage.Define<string>(
-                eventId: 9,
+                eventId: new EventId(9, "EnteringOpenIdAuthenticationHandlerHandleRemoteAuthenticateAsync"),
                 logLevel: LogLevel.Trace,
                 formatString: "Entering {OpenIdConnectHandlerType}'s HandleRemoteAuthenticateAsync.");
             _nullOrEmptyAuthorizationResponseState = LoggerMessage.Define(
-                eventId: 10,
+                eventId: new EventId(10, "NullOrEmptyAuthorizationResponseState"),
                 logLevel: LogLevel.Debug,
                 formatString: "message.State is null or empty.");
             _unableToReadAuthorizationResponseState = LoggerMessage.Define(
-                eventId: 11,
+                eventId: new EventId(11, "UnableToReadAuthorizationResponseState"),
                 logLevel: LogLevel.Debug,
                 formatString: "Unable to read the message.State.");
             _responseError = LoggerMessage.Define<string, string, string>(
-                eventId: 12,
+                eventId: new EventId(12, "ResponseError"),
                 logLevel: LogLevel.Error,
                 formatString: "Message contains error: '{Error}', error_description: '{ErrorDescription}', error_uri: '{ErrorUri}'.");
             _responseErrorWithStatusCode = LoggerMessage.Define<string, string, string, int>(
-                eventId: 49,
+                eventId: new EventId(52, "ResponseErrorWithStatusCode"),
```

Comment:
There was already event id `49` for `SignOut` so I changed this id, which is a breaking change.

Reply:
Funny, I actually used 52 in my previous PR for a new event. In my defence, I didn't know which one would be merged first :D